test/fork/basic-functionality-tests.js

```diff
@@ -483,6 +530,43 @@ describe('basic functionality tests', function () {
     expect(coverCountAfter).to.be.equal(coverCountBefore.add(1));
   });
 
+  it('Buy ybUSDC yield token cover with USDC', async function () {
+    await evm.impersonate(HUGH);
+    const coverBuyer = await getSigner(HUGH);
+    const coverBuyerAddress = await coverBuyer.getAddress();
+
+    const coverAsset = 6; // DAI
```

Comment:
nit: Will USDC cover asset's ID be 6? If so the comment might need to be updated. ☺️